lib/config/presets/internal/group.ts

```diff
@@ -321,6 +321,16 @@ const staticGroups = {
       },
     ],
   },
+  mkdocsAndMaterial: {
```

Comment:
Let me know if you want a different name for the preset!

I can't decide between these:

- `group:mkdocsandmaterial`
- `group:MkDocsAndMaterialForMkDocs`
- `group:mkdocsAndMaterialformkdocs`
- Or maybe you like something else better :smile: